### ISSUE TITLE
Is project still alive ?

### ISSUE THREAD

**Issue by VLARINT3G** [sailfishos-chum/zsh-packaging#2]
Dear, developers. 
Will you maintain your zsh-project in the future ?

**Comment by rinigus** [comment]
Yes it is. As long as there are users and developers that want to take care of it. SFOS has rather small number of users and we have to take care of software that we use. Looks like you care about ZSH and I suggest that you would take care of updating it and testing updates. For many of us, contribution started with smaller projects and why not to take care of this package?
